plugins/org.jkiss.dbeaver.ext.clickhouse/src/org/jkiss/dbeaver/ext/clickhouse/model/ClickhouseSQLDialect.java

```diff
@@ -278,6 +279,8 @@ public String getTypeCastClause(@NotNull DBSTypedObject attribute, String expres
         String typeName = attribute.getTypeName();
         if (isInCondition && CommonUtils.isNotEmpty(typeName) && ClickhouseConstants.DATA_TYPE_IPV4.equals(typeName.toLowerCase())) {
             return "IPv4StringToNum(" + expression + ")";
+        } else if (isInCondition && CommonUtils.isNotEmpty(typeName) && ClickhouseConstants.DATA_TYPE_IPV6.equals(typeName.toLowerCase())) {
+            return "IPv6StringToNum(" + expression + ")";
```

Comment:
Let's rewrite it to avoid checking the same condition `isInCondition && CommonUtils.isNotEmpty(typeName)` twice.

Reply:
Alright, I have made the changes.